app/Services/User/UserService.php

```diff
@@ -46,4 +46,16 @@ public function checkRegisterCourse($course_id)
 
         return $course;
     }
+
+    /**
+     * get list user of sys
+     *
+     * @return mixed
+     */
+    public function getAll()
+    {
+        $users = User::paginate();
```

Comment:
cái này có extends thì baseSevice rồi thì xài nó luôn đi...
$this->paginate();
paginate bao nhiêu thif truyền vô

Reply:
khai báo perPage trong model rôi nên k cần truyền tham sô ở đây.